js/src/main/scala/org/widok/bindings/Bootstrap/package.scala

```diff
@@ -123,9 +130,9 @@ package object Bootstrap {
   }
 
   trait ButtonBase[T] extends Widget[T] { self: T =>
```

Comment:
You could declare ``ButtonBase`` in ``HTML`` and extend from it here. Then you don't need to set the underlying element.

```scala
trait ButtonBase[T] extends HTML.ButtonBase[T] { self: T =>
  ...
}
````